### PR TITLE
support for webpack2

### PR DESCRIPTION
**Pull request by BKrieger73**
@AlexChaseJones upgraded to webpack2. Can you take a look?

run npm install to update locally to webpack2

### CODE REVIEW

**Comment by BKrieger73** [comment]
@dagifted1

**Comment by dagifted1** [comment]
@AlexChaseJones I'm going to PR this since it has breaking changes.

File: package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "style-loader": "^0.16.1",
     "svgo-loader": "^1.2.1",
     "webpack": "^1.15.0",
```
Comment (dagifted1) [inline package.json]: @BKrieger73 still webpack version 1

File: package.json [hunk 2]
```diff
@@ -21,24 +21,24 @@
   },
   "devDependencies": {
     "babel-core": "^6.24.1",
```
Comment (dagifted1) [inline package.json]: @BKrieger73 this is on version 6.25.0

File: package.json [hunk 3]
```diff
@@ -21,24 +21,24 @@
...
-    "babel-loader": "^6.4.1",
+    "babel-loader": "^7.1.0",
     "babel-polyfill": "^6.23.0",
     "babel-preset-es2015": "^6.24.1",
     "clean-webpack-plugin": "^0.1.16",
     "copy-webpack-plugin": "^4.0.1",
     "css-loader": "^0.28.0",
```
Comment (dagifted1) [inline package.json]: @BKrieger73 css-loader is on version 0.28.4

File: package.json [hunk 4]
```diff
@@ -21,24 +21,24 @@
...
-    "eslint": "^3.19.0",
-    "eslint-config-google": "^0.7.1",
+    "eslint": "^4.0.0",
```
Comment (dagifted1) [inline package.json]: @BKrieger73 eslint is on version 4.1 but i think this was published within the last day

File: package.json [hunk 5]
```diff
@@ -21,24 +21,24 @@
...
+    "eslint-config-google": "^0.8.0",
     "eslint-loader": "^1.6.1",
```
Comment (dagifted1) [inline package.json]: @BKrieger73 eslint-loader is on version 1.8.0

File: package.json [hunk 6]
```diff
@@ -21,24 +21,24 @@
...
     "file-loader": "^0.11.1",
```
Comment (dagifted1) [inline package.json]: @BKrieger73 file-loader is on version 0.11.2

File: package-lock.json [hunk 7]
```diff
@@ -0,0 +1,4732 @@
+{
```
Comment (dagifted1) [inline package-lock.json]: @BKrieger73 this lock file shouldn't be committed, should be unlocked/deleted/removed

File: package.json [hunk 8]
```diff
@@ -21,24 +21,24 @@
...
+    "json-loader": "^0.5.4",
```
Comment (dagifted1) [inline package.json]: @BKrieger73 is this still needed? I think it might be bundled with webpack2 now...not sure but test and see if we can remove please

Comment (BKrieger73) [inline package.json]: @dagifted everything online says json-loader is not needed and the starter bundle seems to work without it.

**Comment by BKrieger73** [comment]
@dagifted1 I pushed all of the changes listed in your comments.

**Pull request merged**